avocado/utils/multipath.py

```diff
@@ -106,6 +106,36 @@ def get_multipath_wwids():
     return wwids
 
 
+def get_multipath_wwid(mpath):
+    """
+    Get the wwid binding for given mpath name
+
+    :return: Multipath wwid
+    :rtype: str
+    """
+    cmd = "multipathd show maps format '%%n %%w' | grep %s" % mpath
+    wwid = process.run(cmd, ignore_status=True,
+                        sudo=True, shell=True).stdout_text
+    if wwid:
+        return wwid.split()[1]
```

Comment:
@harihare Please use native python parsing instead of bash parsing
```
cmd = "multipathd show maps format '%%n %%w'
wwid = process.run(cmd, ignore_status=True,
                   sudo=True, shell=True).stdout_text
if mpath in wwid.splitlines():
    return wwid.split()[1]
````

Reply:
Done